src/components/__tests__/NotAuthorisedView.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import { fireEvent } from '@testing-library/react-native'
+import { mocked } from 'jest-mock'
+import React from 'react'
+import { Linking } from 'react-native'
+
+import { getLabels } from '../../services/helpers'
+import render from '../../testing/render'
+import NotAuthorisedView from '../NotAuthorisedView'
+
+jest.mock('react-native/Libraries/Linking/Linking', () => ({
+  openSettings: jest.fn(),
+}))
+
+describe('NotAuthorisedView', () => {
+  const setVisible = jest.fn()
+  it('should render description', () => {
+    const { getByTestId } = render(
+      <NotAuthorisedView setVisible={setVisible} description={getLabels().general.camera.noAuthorization.description} />
+    )
+    expect(getByTestId('no-auth')).toBeTruthy()
+  })
+  it('should successfully go back', () => {
+    const { getByText } = render(
+      <NotAuthorisedView setVisible={setVisible} description={getLabels().general.camera.noAuthorization.description} />
+    )
+    const buttonVisible = getByText(getLabels().general.back)
+    fireEvent.press(buttonVisible)
```

Comment:
This test doesn't have an `expect` statement. Does it run? Can it fail?

Reply:
it ran without error, I will add **expect**